playground/vite-react-ts/src/App.tsx

```diff
@@ -0,0 +1,118 @@
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import clerkLogo from '/clerk.svg'
+import './App.css'
+import { ClerkProvider, RedirectToSignIn, SignIn, SignUp, SignedIn, SignedOut, UserButton, UserProfile, useUser } from "@clerk/clerk-react";
+import { BrowserRouter, Route, Routes, useNavigate } from "react-router-dom";
+
+if (!import.meta.env.VITE_REACT_APP_CLERK_PUBLISHABLE_KEY) {
+  throw new Error("Missing Publishable Key")
+}
```

Comment:
❓ wouldn't the `ClerkProvider` raise an error if no publishableKey or frontendApi exists? I think we should omit this check.

Reply:
We don't need this check

Reply:
Removed!